Source/ModuleSimpleBoiloff.cs

```diff
@@ -378,14 +378,10 @@ public void Update()
                   double max = GetTotalMaxResouceAmount();
 
                   CoolingStatus =  Localizer.Format("#LOC_CryoTanks_ModuleCryoTank_Field_CoolingStatus_Editor", (CoolingCost * (float)(max / 1000.0)).ToString("F2"));
-              }
-              if (CoolingCost > 0f && !hasResource)
-                Fields["CoolingStatus"].guiActive = false;
```

Comment:
Won't removing this check cause resources that are specified in the cooling config but not present on the part to keep their enable/disable buttons?